pyproject.toml

```diff
@@ -16,6 +16,9 @@ dependencies = [
 requires-python = ">=3.0"
 authors = [
   {name = "Pavel Raiskup", email = "praiskup@redhat.com"},
+  {name = "Slávek Kabrda", email = "bkabrda@redhat.com"},
+  {name = "Tomáš Tomeček", email = "ttomecek@redhat.com"},
+  {name = "Ales Nezbeda", email = "anezbeda@redhat.com"},
```

Comment:
Fair point, with that removal of `AUTHORS` file is also preferred. I think it is redundant anyway since the authorship is contained in the module metadata or in other places like github.